doctrine/reverse_engineering.rst

```diff
@@ -1,15 +1,443 @@
+        public function setStr1(?string $str1): static
+        {
+            $this->str1 = $str1;
+
+            return $this;
+        }
+
+        public function getStr2(): ?string
+        {
+            return $this->str2;
+        }
+
+        public function setStr2(?string $str2): static
+        {
+            $this->str2 = $str2;
+
+            return $this;
+        }
+
+        public function getStr3()
+        {
+            return $this->str3;
+        }
+
+        public function setStr3($str3): static
+        {
+            $this->str3 = $str3;
+
+            return $this;
+        }
+
+        public function getStr4()
+        {
+            return $this->str4;
+        }
+
+        public function setStr4($str4): static
+        {
+            $this->str4 = $str4;
+
+            return $this;
+        }
+
+        public function getStr5()
+        {
+            return $this->str5;
+        }
+
+        public function setStr5($str5): static
+        {
+            $this->str5 = $str5;
+
+            return $this;
+        }
+
+        public function getStr6(): ?string
+        {
+            return $this->str6;
+        }
+
+        public function setStr6(?string $str6): static
+        {
+            $this->str6 = $str6;
+
+            return $this;
+        }
+
+        public function getStr8(): ?array
+        {
+            return $this->str8;
+        }
+
+        public function setStr8(?array $str8): static
+        {
+            $this->str8 = $str8;
+
+            return $this;
+        }
+
+        public function getJson1(): ?array
+        {
+            return $this->json1;
+        }
+
+        public function setJson1(?array $json1): static
+        {
+            $this->json1 = $json1;
+
+            return $this;
+        }
+
+        public function getBool1(): ?int
+        {
+            return $this->bool1;
+        }
+
+        public function setBool1(?int $bool1): static
+        {
+            $this->bool1 = $bool1;
+
+            return $this;
+        }
+    }
+
 .. caution::
 
-    The ``doctrine:mapping:import`` command used to generate Doctrine entities
-    from existing databases was deprecated by Doctrine in 2019 and there's no
-    replacement for it.
+    The ``--ucfirst=true`` option is used to ensure compatibility with private properties in entities from Symfony 6.0, so that you can seamlessly migrate from Symfony 6.0 to 7.0 without modifying your business code due to changes in entity descriptions. Refer to the `doctrine-helper`_ documentation for more command parameters.
 
-    Instead, you can use the ``make:entity`` command from `Symfony Maker Bundle`_
-    to help you generate the code of your Doctrine entities. This command
-    requires manual supervision because it doesn't generate entities from
-    existing databases.
+The generated entities are now ready to be used. Have fun!
 
-.. _`Symfony Maker Bundle`: https://symfony.com/bundles/SymfonyMakerBundle/current/index.html
```

Comment:
why not keep both, and just in your repo project's readme add such documentation ?
i do not think this is relevant to document much here as the tool is not maintained by sf core ppls, thus maintaining a synchro in code+doc can be very hard here

Reply:
<img width="789" alt="image" src="https://github.com/symfony/symfony-docs/assets/15135780/e5fc4749-a4c9-46ce-ad13-b225dc941d1a">

Can this be changed?